run.go

```diff
@@ -46,45 +42,6 @@ func (bf *BuildFactory) RunConfigFromFlags(ctx context.Context, f *RunFlags) (*R
 	return rc, nil
 }
 
-func Run(ctx context.Context, outWriter, errWriter io.Writer, appDir, buildImage, runImage string, ports []string) error {
```

Comment:
I took the liberty of removing this as it seemed unused and duplicate of what the command does. Can revert if I was a bit hasty.

Reply:
Reverted as I just realized this is intended to be used as a library as well.